packages/foam-vscode/src/web/features/preview/wikilink-embed.ts

```diff
@@ -0,0 +1,12 @@
+/*global markdownit:readonly*/
+
+import { ResourceParser } from 'packages/foam-vscode/src/core/model/note';
+import { FoamWorkspace } from 'packages/foam-vscode/src/core/model/workspace';
+
+export const markdownItWikilinkEmbed = (
+  md: markdownit,
+  workspace: FoamWorkspace,
+  parser: ResourceParser
+) => {
+  return md;
```

Comment:
I would actually return something along the lines of:
```
return `<div class="foam-no-embeds-warning">Embeds are not supported in web extension: ${wikilink}</div>`;
```

`foam-no-embeds-warning` could be along the lines of `foam-cyclic-link-warning`

This is gonna save us from a bunch of bug reports about the missing feature ;)

Reply:
<img width="348" alt="Screenshot 2024-07-11 at 20 18 57" src="https://github.com/foambubble/foam/assets/495374/e818388b-c35f-4c72-a153-7c6f22e30e61">